src/styles/card.scss

```diff
@@ -0,0 +1,73 @@
+.card {
+  width: 198px; // including border
+  height: 406px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    padding: 32px 19px 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+    color: $card-title-color;
+    padding-bottom: 4px;
+    padding-left: 16px;
+    padding-right: 16px;
+  }
+
+  &__cod {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 140%;
+    color: $card-code-color;
+    padding-bottom: 16px;
+    padding-left: 16px;
+    padding-right: 16px;
```

Comment:
Similar to the previous comment, consider using padding shorthand for the `&__cod` class as well.